### ISSUE TITLE
Je m'attendais à moins de click

### ISSUE THREAD

**Issue by vitoo** [jmleroux/covid-attestation#22]
Bonjour,

Merci d'avoir codé ça !
La seule en php que j'ai trouvé

Je pense qu'il aurait fallut sauvegarder aussi les justifications de déplacement, j'aurais voulu avoir une URL qu'on met sur la homepage du téléphone, et qui télécharge directement le pdf sur le téléphone en 1 click

Car le lien du gouv propose le même nombre de click (on doit click sur la justification, après avoir sauvegarder les informations)

**Comment by jmleroux** [comment]
> Car le lien du gouv propose le même nombre de click (on doit click sur la justification, après avoir sauvegarder les informations)

Tout à fait, ils ont amélioré le process et proposent maintenant une solution tout à fait viable.

**Comment by vitoo** [comment]
Oui

s'il y a un nouveau confinement, je voudrais bien une solution avec 1 seul click pour gagner du temps, je ferais un petit fork :)

**Comment by vitoo** [comment]
Bon j'ai forké,  j'ai mis l'étape des justification dans la 1ere, par contre j'ai fait ça rapidement, pas d'update de tests etc, donc pas prêt à faire une pull request

merci pour ton code en tout cas ! ça risque de m'être utile malheureusement 😢
